### PR TITLE
Docs pass

### PR DESCRIPTION
**Pull request by kylebarron**
Initial docs cleanup.

I'd suggest a lot of work on the `v2_migration` document; much of that can be moved to other files. That file should be small and concise with only the things the user needs to know to get their code to work on v2

### CODE REVIEW

**Comment by vincentsarago** [comment]
🎉

File: docs/mosaic.md [hunk 1]
```diff
@@ -17,10 +23,11 @@ Because user might want to choose which pixel goes on top of the tile, this plug
 `rio_tiler.mosaic.mosaic_reader(assets, tiler, *args* pixel_selection=None, chunk_size=None, Threads=10, **kwargs)`
 
 Inputs:
+
 - assets : list, tuple of rio-tiler compatible assets (url or sceneid)
-- tiler: Rio-tiler's tiler function (e.g rio_tiler.landsat8.tile) 
+- tiler: Rio-tiler's tiler function (e.g rio_tiler.landsat8.tile)
```
Comment (vincentsarago) [inline docs/mosaic.md:28]: can we replace this by `tiler: Callable that returns a tuple of numpy.array (e.g tile, mask = rio_tiler.reader.tile(x, y, z, **kwargs)`

File: docs/v2_migration.md [hunk 2]
```diff
@@ -310,7 +322,7 @@ img = rio_tiler.utils.render(tile, mask, img_format="PNG")
 Each `rio_tiler.io.{mission}` **scene id parser** (e.g cbers_parser) has been refactored and now return AWS S3 path information.
 
 ```python
-rio_tiler.io.landsat8.landsat_parser("LC08_L1TP_016037_20170813_20170814_01_RT")) 
+rio_tiler.io.landsat8.landsat_parser("LC08_L1TP_016037_20170813_20170814_01_RT"))
```
Comment (vincentsarago) [inline docs/v2_migration.md:325]: We can remove this section or just add a note about the new rio-tiler-pds

Comment (kylebarron) [reply docs/v2_migration.md:325]: Yeah I was going to leave `rio-tiler-pds` for the next docs PR

**Pull request merged**